README.md

```diff
@@ -46,6 +46,14 @@ For macOS:
 @import "custom/_megabar-disable.css";
 ```
 
+### Add Developer Tools styling
+
+Set theme to dark in Firefox Developer Tools [settings](https://developer.mozilla.org/en-US/docs/Tools/Settings). Then, add the following line to `userContent.css`:
+
+```
+@import "_dev-tools.css";
```

Comment:
Since this functionality is marked as optional and is not part of the main core of the theme, this file should reside in the custom folder.